{{cookiecutter.package_name}}/environment.sh

```diff
@@ -88,26 +89,37 @@ else
     jq_exists=$(conda list | grep -w jq)
     if [[ $jq_exists == '' ]]; then
       # Empty string is no return on grep
-      conda install jq -y
+      conda install jq -c anaconda -y
     fi
+    echo "Checking framework packages are up to date..."
     # Check if there has been an update to vivarium packages since last modification to requirements file
     # or more reccent than environment creation
-    grep @ $install_file 
-    # TODO: can we make this grep output a variable?
-    while read -r line ; do
-      # Parse each line of grep output
-      repo_info=(${line//@/ })
-      repo=${repo_info[0]}
-      repo_branch=${repo_info[2]}
-      last_commit_time=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/ihmeuw/$repo/commits?sha=$repo_branch | jq '.["0"].commit.committer.date')
-      if [[ $creation_time > $last_commit_time ]]; then
+    # Note: The lines we will return via grep will look like 'vivarium>=#.#.#' or will be of the format 
+    # 'vivarium @ git+https://github.com/ihmeuw/vivarium@SOME_BRANCH'
+    # echo $(grep -E 'vivarium|gbd|risk_distribution|layered_config' $install_file)
+    framework_packages=$(grep -E 'vivarium|gbd|risk_distribution|layered_config' $install_file)
```

Comment:
the "vivarium" will also catch any `vivarium_*` packages. Does that matter or is that intended?

Reply:
Yes it will. Anything with vivarium in the line